### PR TITLE
Added multidelete function for Service Accounts

### PR DESCRIPTION
**Pull request by jinapurapu**
**What does it do**: Adds multidelete function for Service Accounts on Service Account screen and User Service Account tab
Closes #469

Screenshot from 2022-02-01 11-43-33
Screenshot from 2022-02-01 11-42-59
Screenshot from 2022-02-01 11-44-17

### CODE REVIEW

**Review by adfost — CHANGES_REQUESTED** [review]
Dependencies don't compile

**Comment by bexsoft** [comment]
> Dependencies don't compile

@adfost Please review your local configuration, backend compiles on my end.

File: portal-ui/src/screens/Console/Configurations/utils.tsx [hunk 1]
```diff
@@ -522,3 +522,24 @@ export const removeEmptyFields = (formFields: IElementValue[]) => {
...
+
+export const selectSAs = (e: React.ChangeEvent<HTMLInputElement>, setSelectedSAs : Function, selectedSAs : string[]) => {
+  const targetD = e.target;
+  const value = targetD.value;
+  const checked = targetD.checked;
+console.log("You are in the util function")
+console.log("targetD is :", targetD)
+console.log("value is :", value)
+console.log("checked is :", checked)
+console.log("selecteSAS is :", selectedSAs)
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Configurations/utils.tsx:534]: Please remove these console.logs

```suggestion

```

File: portal-ui/src/screens/Console/Users/DeleteMultipleServiceAccounts.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        </DialogContentText>
+      }
+    />
+  );
+};
+const mapDispatchToProps = {
+  setErrorSnackMessage,
+};
+const connector = connect(null, mapDispatchToProps);
+export default connector(DeleteMultipleSAs);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/DeleteMultipleServiceAccounts.tsx:72]: Please add empty space at the end of the file
```suggestion
export default connector(DeleteMultipleSAs);

```

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by dvaldivia — APPROVED** [review]
Tested. LGTM. After addressing the comments above will re-check.

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**